### PR TITLE
mv jhoblitt, igoodenow, jcockrum, & shahram accounts to ipa

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

File: site/tu.yaml [hunk 1]
```diff
@@ -9,6 +9,16 @@
 # InfluxDB Master               : 140.252.32.159
 # Grafana                       : 140.252.32.158
 # Puppet Master                 : 140.252.32.139
+lookup_options:
```
Comment (adrienthebo) [inline site/tu.yaml]: Is it reasonable and/or appropriate to move this to common.yaml?

Comment (jhoblitt) [reply site/tu.yaml]: Probably?  I intentionally didn't as I'm unsure if we're going to end up with with other non-aura sites in the tree but I'm willing to revise it.

Comment (adrienthebo) [reply site/tu.yaml]: Let's punt on this.

Comment (jhoblitt) [reply site/tu.yaml]: That works for me.

**Pull request merged**